packages/tradershub/src/features/cfd/screens/CFDSuccess/CFDSuccess.tsx

```diff
@@ -10,11 +10,14 @@ import { TMarketTypes, TPlatforms } from '../../../../types';
 
 type TProps = {
```

Comment:
Change it to `TCFDSuccess` atleast we know from first glance what prop is this for